package.json

```diff
@@ -44,26 +44,26 @@
     "extends": "enact-proxy/strict"
   },
   "dependencies": {
-    "@enact/core": "^4.5.0",
-    "@enact/i18n": "^4.5.0",
-    "@enact/spotlight": "^4.5.0",
-    "@enact/ui": "^4.5.0",
-    "classnames": "^2.3.1",
+    "@enact/core": "^4.5.2",
+    "@enact/i18n": "^4.5.2",
+    "@enact/spotlight": "^4.5.2",
+    "@enact/ui": "^4.5.2",
+    "classnames": "^2.3.2",
     "invariant": "^2.2.4",
     "prop-types": "^15.8.1",
     "ramda": "^0.28.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "warning": "^4.0.3",
-    "ilib": "14.14.0"
+    "ilib": "^14.15.1"
```

Comment:
As I commented on agate PR, please pin this version to 14.14.0.

Reply:
Hello @MikyungKim , as Daniel mentioned in https://github.com/enactjs/agate/pull/687. Ilib team actually released 14.15.1 a few days after enact team created an issue. The isomorphic build failure issue is fixed https://github.com/iLib-js/iLib/releases/